sound/soc/sof/ipc4-topology.c

```diff
@@ -968,6 +968,14 @@ sof_ipc4_update_resource_usage(struct snd_sof_dev *sdev, struct snd_sof_widget *
 	pipe_widget = swidget->spipe->pipe_widget;
 	pipeline = pipe_widget->private;
 	pipeline->mem_usage += total;
+
+	/* Update base_config->cpc from the module manifest */
+	sof_ipc4_update_cpc_from_manifest(sdev, swidget->module_info, base_config);
+
+	dev_dbg(sdev->dev, "ibs / obs / cpc for %s: %u / %u / %u\n",
+		swidget->widget->name, base_config->ibs, base_config->obs,
```

Comment:
I still don't get how we deal with a module that does NOT have an input buffer or pin.